src/CheckoutSdk/Accounts/Document.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Checkout.Accounts
+﻿using Checkout.Common;
+
+namespace Checkout.Accounts
```

Comment:
Can we review this class because it's under the Accounts folder, and If we want to use it on common, we should move also the file

Reply:
This is correct, because Document depends of Account, inside Document the enum DocumentType has moved to common because Issuing use it.